core/optaplanner-core-impl/src/main/java/org/optaplanner/core/impl/domain/solution/cloner/DeepCloningUtils.java

```diff
@@ -77,17 +69,6 @@ public final class DeepCloningUtils {
             HardMediumSoftScore.class, HardMediumSoftLongScore.class, HardMediumSoftBigDecimalScore.class,
             BendableScore.class, BendableLongScore.class, BendableBigDecimalScore.class);
 
-    private final ConcurrentMap<Pair<Field, Class<?>>, Boolean> fieldDeepClonedMemoization = new ConcurrentMemoization<>();
-    private final ConcurrentMap<Class<?>, Boolean> actualValueClassDeepClonedMemoization = new ConcurrentMemoization<>();
-    private final SolutionDescriptor<?> solutionDescriptor;
-
-    public DeepCloningUtils(SolutionDescriptor<?> solutionDescriptor) {
-        this.solutionDescriptor = solutionDescriptor;
-    }
-
```

Comment:
This is purely a util class now. Does not have any instance fields, does not do any caching.
The caching moved to `FieldAccessingSolutionCloner`.
Caching was entirely removed from GIZMO, as - if my understanding is correct - all of those operations are done at build-time, and therefore do not require the cache.